examples/with-pipes/main.tf

```diff
@@ -673,3 +687,58 @@ module "kms" {
 
   key_owners = [data.aws_caller_identity.current.arn]
 }
+
+####################
+# MSK
+####################
+resource "aws_vpc" "main" {
```

Comment:
lets use our modules to show this https://github.com/terraform-aws-modules/terraform-aws-msk-kafka-cluster/blob/master/examples/basic/main.tf

also, lets add a variable to enable/disable the MSK resources/integration. provisioning a cluster takes quite a bit of time 😅